openedx/adg/lms/applications/management/commands/tests/test_update_is_prerequisite_courses_passed.py

```diff
@@ -0,0 +1,175 @@
+"""
+Tests to check if update_is_prerequisite_courses_passed command
+actually updates the is_prerequisite_courses_passed flag in
+Application Hub for each eligible user
+"""
+
+from datetime import datetime
+
+import pytest
+from django.core.management import call_command
+
+from openedx.adg.common.course_meta.factory import CourseMetaFactory
+from openedx.adg.common.course_meta.models import CourseMeta
+from openedx.adg.lms.applications.management.commands import update_is_prerequisite_courses_passed as command_module
+from openedx.adg.lms.applications.models import ApplicationHub
+from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
+from student.tests.factories import CourseEnrollmentFactory, UserFactory
+
+
+@pytest.fixture
+def user_for_testing(request):
+    return UserFactory.create()
+
+
+@pytest.fixture
+def course_for_testing(request):
+    prerequisite_course = CourseOverviewFactory.create(
+        end_date=datetime(2022, 1, 1),
+        start_date=datetime(2015, 1, 1),
+        display_name='pre requisite course'
+    )
+
+    CourseMetaFactory(course=prerequisite_course)
+
+    return prerequisite_course
+
+
+@pytest.fixture
+def enroll_user(request, user_for_testing, course_for_testing):
+    CourseEnrollmentFactory.create(
+        user=user_for_testing,
+        course_id=course_for_testing.id,
+        is_active=request.param['is_active']
+    )
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize('enroll_user',
+                         [dict({'is_active': False})],
+                         indirect=True)
+def test_minimal_users_with_inactive_user(enroll_user):
+    pre_req_courses = CourseMeta.open_pre_req_courses.all()
```

Comment:
Please add a short docstring for each unit test. For unit test docstring we do not add params or return type in docstring.

```
"""
My test docstring
"""
```